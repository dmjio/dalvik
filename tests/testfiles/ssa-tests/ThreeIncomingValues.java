class ThreeIncomingValues {
  static int entry(int x) {
    int z;
    switch(x) {
    case 1:
      z = 5;
      break;
    case 2:
      z = 11;
      break;
    default:
      z = 0;
    }

    return z;
  }
}
