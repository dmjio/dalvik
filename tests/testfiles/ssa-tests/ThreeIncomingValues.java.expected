SimplePhi { numBlocks = 5
          , phis = [ (0, fromList ["0", "5", "11"]) ]
          , blockPhis = [(0, 2)]
          , assignedPhis = []
          }
